test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py

```diff
@@ -76,6 +78,10 @@ def test_eks_pytorch_dgl_single_node_training(pytorch_training, py3_only):
     Args:
         :param pytorch_training: the ECR URI
     """
+    _, image_framework_version = get_framework_and_version_from_tag(pytorch_training)
+    image_cuda_version = get_cuda_version_from_tag(pytorch_training)
+    if Version(image_framework_version) == Version("1.6") and image_cuda_version == "cu110":
+        pytest.skip("DGL does not suport CUDA 11 for PyTorch 1.6")
```

Comment:
Does this test still need to be skipped? DGL has been installed using a custom-built wheel, right?

Reply:
Even with custom build this issue is noted